python/ray/_private/services.py

```diff
@@ -1929,3 +1929,22 @@ def start_ray_client_server(redis_address,
         stderr_file=stderr_file,
         fate_share=fate_share)
     return process_info
+
+
+def get_object_locations(obj_refs, timeout_ms=-1):
+    """Get object locations.
+
+    Returns:
+        A list of ObjectLocations for each Object. ObjectLocation could
```

Comment:
Can it return a dict from the object id -> location?

Reply:
sure, that works!